src/repository.py

```diff
@@ -10,6 +10,7 @@ class Repository:
 	def connect_db():
 		return sqlite3.connect(Repository.DBPATH)
 
+
 	@staticmethod
 	def add_loan_account(loan_amt, time_period, interest_rate, account_num):
 		try:
```

Comment:
the variable name can be improved at line 19. Follow `snake_case` for naming a variable.

![image](https://user-images.githubusercontent.com/70574489/217151083-72b55a5f-b5be-4086-8269-76f1119b7e17.png)